### ISSUE TITLE
Versioning of quickMockup

### ISSUE THREAD

**Issue by simonv3** [jdittrich/quickMockup#31]
It could be that at some point the structure of quickMockup gets changed so much that an existing quickMockup file might not work anymore with the current version. Should we keep track of versioning to detect what might be breaking changes? Eg. [semver](http://semver.org/)?

**Comment by jdittrich** [comment]
The loading of an existing mockup was recently implemented. Before the assumption was that the user just save the whole page, thus "breaking" did not make much sense before. 

Now it might, but the function is still too new (3/2016) to say how often such changes occur. 

I hope to remember this for the next change.

**Comment by simonv3** [comment]
Yeah, it's not really an issue now, more of an issue later, remembering it for later is why I created the issue :)
